.vib/consul/goss/goss.yaml

```diff
@@ -33,11 +23,3 @@ command:
       - "consul-1"
       - "consul-2"
```

Comment:
Can we use the `range` feature here to render this dynamically based on `replicaCount`?